codegen/generate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mvn clean package -DskipTests
+java -cp target/codegen-1.0.0-SNAPSHOT-shaded.jar org.nd4j.codegen.cli.CLI -dir ../../deeplearning4j -namespaces "$@"
```

Comment:
Can you also add a a sentence or two and an example under the usage section in the readme?

Reply:
Done